src/api/app/views/webui2/webui/groups/show.html.haml

```diff
@@ -0,0 +1,56 @@
+- @pagetitle = "Group #{@group.title}"
+
+.card
+  %div
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+      %li.nav-item
+        %a.active.nav-link.text-nowrap{ aria: { controls: 'reviews-in', selected: 'true' },
+        data: { toggle: 'tab' }, href: '#reviews-in', role: 'tab' }
+          Incoming Reviews
+      %li.nav-item
+        %a.nav-link.text-nowrap{ aria: { controls: 'requests-in' }, data: { toggle: 'tab' }, href: '#requests-in', role: 'tab' }
+          Incoming Requests
+      %li.nav-item
+        %a.nav-link.text-nowrap{ aria: { controls: 'all-requests' }, data: { toggle: 'tab' }, href: '#all-requests', role: 'tab' }
+          All Requests
+      %li.nav-item
+        %a.nav-link.text-nowrap{ aria: { controls: 'group-members' }, data: { toggle: 'tab' }, href: '#group-members', role: 'tab' }
+          Group Members
+
+  .card-body.tab-content
+    .tab-pane.show.active#reviews-in{ aria: { controls: 'reviews-in' }, role: 'tabpanel' }
+      %h3 Incoming Reviews
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#requests-in{ aria: { controls: 'requests-in' }, role: 'tabpanel' }
+      %h3 Incoming Requests
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'requests_in_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#all-requests{ aria: { controls: 'all-requests' }, role: 'tabpanel' }
+      %h3 All Requests
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'all_requests_table', source_url: group_requests_path(@group) })
+
+    .tab-pane#group-members{ aria: { controls: 'group-members' }, role: 'tabpanel' }
+      %h3 Group Members
+      %table.responsive.table.table-striped.table-bordered#group-members-table{ width: '100%' }
+        %thead
+          %tr
+            %th Login Name
+        %tbody
+          - if @group.users.any?
```

Comment:
To see that there is no content? :thinking:

I think I would find it confusing if the table some times is there and sometimes it's not.  Since that's the main job of this tab, showing the group members.

Reply:
This is how it will look like with the current code for a group without users:

![group_members_empty](https://user-images.githubusercontent.com/24919/49281286-79bf9e80-f48c-11e8-921d-1ede7fee4149.png)

I'm with @bgeuken.

Reply:
I agree with @bgeuken and @eduardoj

Reply:
We do it differently in other pages such as the user home page page. We need some application consistency here.